katello_certs_tools/katello_ssl_tool.py

```diff
@@ -645,15 +684,14 @@ def genCaRpm(d, verbosity=0):
     fo.write('%s.noarch.rpm\n' % os.path.basename(clientRpmName))
     fo.write('%s.src.rpm\n' % os.path.basename(clientRpmName))
     fo.close()
-    os.chmod(latest_txt, 0644)
+    os.chmod(latest_txt, '0644')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)